images/tf_operator/build_and_push.py

```diff
@@ -106,16 +106,24 @@ def main():  # pylint: disable=too-many-locals, too-many-statements
   targets = [
       "github.com/tensorflow/k8s/cmd/tf_operator",
       "github.com/tensorflow/k8s/test/e2e",
+      "github.com/tensorflow/k8s/dashboard/backend",
   ]
   for t in targets:
     subprocess.check_call(["go", "install", t])
+  
+  # Resolving dashboard's front-end dependencies
+  subprocess.check_call(["yarn", "--cwd", "./dashboard/frontend", "install"])
```

Comment:
You also need to update:
https://github.com/tensorflow/k8s/blob/master/py/release.py
This is what the tests use and I think that's why the presubmit is failing.